app/assets/yge/youvegoteth/shared.js

```diff
@@ -0,0 +1,47 @@
+var retrieveAmount = function () {
+    var ele = jQuery("input[amount]");
+    var target_ele = jQuery("#usd_amount");
+
+    if (target_ele.html() == "") {
+        target_ele.html('<img style="width: 50px; height: 50px;" src=/static/v2/images/loading.gif>');
+    }
+
+    var amount = jQuery("input[name=amount]").val();
+    var denomination = "ETH";
```

Comment:
any way to make the module correctly handle non-ETH tips
<img width="440" alt="screen shot 2017-12-03 at 9 12 05 pm" src="https://user-images.githubusercontent.com/513929/33536019-aa7cc4ac-d86e-11e7-8a98-6d25aeeec01b.png">

hint: if metamask is on the mainnet, a bunch of diff ERC20 tokens will show

Reply:
oh shoot i just realized the 'new bounty' page doesnt really handle this correctly eeither

Reply:
per slack convo   im gonna merge your PR and create a new bounty for ERC20 token compatibility